### ISSUE TITLE
Research and create experiments with different loss functions

### ISSUE THREAD

**Issue by jannikjw** [jannikjw/one-shot-face-recognition#17]
Can you fine-tune a model with a different loss function than it was pre-trained on.
